R/fb_league_stats.R

```diff
@@ -32,26 +32,27 @@
   } else {
 
     rlang::inform(
-      'Please be aware that `fb_league_stats(..., team_or_player = "player")` depends on promises, which may not always work.',
+      'Please be aware that `fb_league_stats(..., team_or_player = "player")` depends on `rvest::read_html_live` (and chromote), which may not always work.',
       .frequency = "once",
       .frequency_id = "fb_league_stats-player"
     )
 
-    session <- worldfootballr_chromote_session(url)
-    player_table <- worldfootballr_html_player_table(session)
-    session$session$close(wait_ = FALSE)
+    page <- rvest::read_html_live(url)
```

Comment:
wow, so easy now! (i can completely delete `chromote-fbref.R` 😄)

Reply:
Have been using `rvest::read_html_live()` for work quite a bit, just didn't get to addressing it here though - awesome stuff!